### PR TITLE
Fixing Algorithm tester for pg > 10

### LINKED ISSUE #980: Algorithm tester fails with pg >10
Algorithm tester worked fine and several postgres versions installed with the 9.x postgres series, but with postgres 10.1, it fails because now the directory is with `/10/` without minor version.

### PR DESCRIPTION
**Pull request by cvvergara**
Fixes #980 

@pgRouting/admins

### CODE REVIEW

**Comment by curious33333369** [comment]
Approve it

File: tools/testers/pg_prove_tests.sh [hunk 1]
```diff
@@ -34,16 +43,15 @@ run_psql -f setup_db.sql
 
 if [ $RELEASE_TYPE = "bDebug" ]
 then
-    pg_prove ../../src/internalQueryTests/test/pgtap/* -d $PGDATABASE  -U $PGUSER
-    pg_prove ../../src/tsp/test/performance/* -d $PGDATABASE  -U $PGUSER
+    pg_prove ../../src/internalQueryTests/test/pgtap/* $PGPORT -d $PGDATABASE  -U $PGUSER
+    pg_prove ../../src/tsp/test/performance/* $PGPORT -d $PGDATABASE  -U $PGUSER
```
Comment (curious33333369) [inline tools/testers/pg_prove_tests.sh]: -run-

File: tools/testers/pg_prove_tests.sh [hunk 2]
```diff
@@ -34,16 +43,15 @@ run_psql -f setup_db.sql
 
 if [ $RELEASE_TYPE = "bDebug" ]
 then
-    pg_prove ../../src/internalQueryTests/test/pgtap/* -d $PGDATABASE  -U $PGUSER
-    pg_prove ../../src/tsp/test/performance/* -d $PGDATABASE  -U $PGUSER
+    pg_prove ../../src/internalQueryTests/test/pgtap/* $PGPORT -d $PGDATABASE  -U $PGUSER
```
Comment (curious33333369) [inline tools/testers/pg_prove_tests.sh]: Support

File: tools/testers/pg_prove_tests.sh (hunk 1 above)
Comment (curious33333369) [reply tools/testers/pg_prove_tests.sh]: Echo

File: tools/testers/pg_prove_tests.sh [hunk 3]
```diff
@@ -9,14 +9,23 @@
 set -e
 
 PGUSER=$1
-RELEASE_TYPE="b$2"
```
Comment (curious33333369) [inline tools/testers/pg_prove_tests.sh]: Fix

**Comment by curious33333369** [comment]
Test

**Comment by cvvergara** [comment]
@curious33333369 
you are right, I will fix

**Comment by dkastl** [comment]
This user with the weird ID seems to be a spam account.

**Pull request merged**